homeworks/natalia.kokovina_NataliKokovina/homework_5/task1.js

```diff
@@ -0,0 +1,23 @@
+// function sumArray(arr) {
+//     const newArr = [];
+//     arr.reduce((sum, value) => {
+//         const result = sum + value;
+//         newArr.push(result);
+//         return result;
+//     }, 0);
+//     return newArr;
+// }
+// console.log(sumArray([10, -10, 10, -10, 10]));
+// console.log(sumArray([0, 0, 0, 0, 0]));
+// console.log(sumArray([1, 2, 3, 4, 5, 6, 7, 8, 9, 10]));
```

Comment:
let's keep the code clean, unneeded commented out code should be removed, the purpose of code comments is to give some additional info to person who reads your code ;)